src/components/DeleteMovieDialog/DeleteMovieDialog.scss

```diff
@@ -0,0 +1,71 @@
+@import "styles/colors";
+@import "styles/fonts";
+@import "styles/buttons";
+
+.delete-movie__container {
+  position: fixed;
+  width: 100%;
+  height: 1787px;
+  left: 0px;
+  top: 0px;
+  z-index: 1;
```

Comment:
A good solution is to have z-index variables stored in one file so it's easy to keep track of what should go above what, like:

```scss
$z-index-dropdown: 10;
$z-index-modal: 100;
$z-index-notification: 1000;
```